app/subscriber/src/features/content/view-content/ViewContent.tsx

```diff
@@ -81,7 +81,7 @@ export const ViewContent: React.FC = () => {
 
   return (
     <styled.ViewContent>
-      <ViewContentToolbar />
+      <ViewContentToolbar currentContent={content && content} />
```

Comment:
Not sure what the logic here means?

Reply:
Passing down the content that is being viewed so the toolbar has access to actions, tags, etc. Have to ensure that content is defined before doing so,  otherwise it will error out

Reply:
This doesn't make sense.   Sounds like you're simply writing code to bypass the linter.  Think about the runtime process.  It's going to pass `undefined` if `content is falsy`

Reply:
I believe if content is falsy the right hand side of the expression will not be passed. Will double check

Reply:
May have just been a bad approach, I was expecting content to be potentially undefined depending on the fetch being executed, and the prop to be updated when that content was defined. Updated the solution.